Modules/LinearBoltzmanSolver/GroupSet/lbs_groupset.h

```diff
@@ -72,6 +74,17 @@ class LBSGroupset
 
   double                                       latest_convergence_metric;
 
+  /**Convenient typdef for the moment call back function. See moment_callbacks*/
+  typedef std::function<void(int, int, int, double)> MomentCallbackF;
```

Comment:
The only part I am missing here is what the moment arguments are. We have two ways to address it. We could either include them into the typedef:

`typedef std::function<void(int dof, int moment, int angle_num, double psi)> MomentCallbackF;`

or we could include it in its doc-string.
`/**Convenient typdef for the moment call back function. See moment_callbacks.
Arguments are:
int dof_mapping Local DOF-address with moment and group folded in.
int moment         The moment number.
int angle_num    The reference angle number in the angular quadrature.
double psi          Angular flux for the given DOF.
*/`

In any case, we will make improvements later so just choose one.

Reply:
I went ahead and updated the comment and added names to the argument list